extensions/lifecycle/tasks/LifecycleTaskV2.js

```diff
@@ -115,8 +118,9 @@ class LifecycleTaskV2 extends LifecycleTask {
                     details: {
                         beforeDate: params.BeforeDate,
                         prefix: params.Prefix,
+                        storageClass: params.ExcludedDataStoreName,
                         listType,
-                        ...markerInfo,
+                        ...markerInfo
```

Comment:
probably an unwanted change (I recall the linter being picky about missing trailing commas)